app/assets/javascripts/nilavu/components/user-account-chngpwd.js.es6

```diff
@@ -1,16 +1,96 @@
 import { propertyEqual } from 'nilavu/lib/computed';
+import debounce from 'nilavu/lib/debounce';
```

Comment:
Can your format it CTRL_ALT_B?
also rename the file to `change-account-password` ?